atlp-rwanda/phantom-fe-dream-team
# Ch setup docker

#### What does this PR do?

SETUP DOCKER

#### Description of Task to be completed?

-An image of the backend application has been created
-An image of the frontend application has been created
-Database is running in a docker container
-Database is running in a docker container

How should this be manually tested?

-Firstly you need to have docker installed
-open your terminal and run: npm run dockercompose
-this will build the image and start the container
-use the docker ps command to see running containers' information
you can see that I have them running on my laptop

check the app on the port 3000 of the local host



Any background context you want to provide?

-N/A

What are the relevant pivotal tracker stories?

![Screenshot (89)](https://user-images.githubusercontent.com/51451731/175348831-81b07d71-c8c1-431f-b9c6-3f83b5b3bea8.png)
